Index: templates/partials/categoryListItem.hbs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><li class=\"header-nav-item\"><a href=\"#\" class=\"header-nav-item-link\">{{word}}</a></li>
===================================================================
diff --git a/templates/partials/categoryListItem.hbs b/templates/partials/categoryListItem.hbs
--- a/templates/partials/categoryListItem.hbs	
+++ b/templates/partials/categoryListItem.hbs	
@@ -1,1 +1,1 @@
-<li class="header-nav-item"><a href="#" class="header-nav-item-link">{{word}}</a></li>
\ No newline at end of file
+<li class="header-nav-item"><a href="#" class="header-nav-item-link">{{this}}</a></li>
\ No newline at end of file
Index: templates/partials/sidebar.hbs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><div class=\"sidebar-container\">\r\n<nav class=\"menu-nav\" aria-label=\"menu\">\r\n    <button class=\"closeBtn\">\r\n        Close menu\r\n    </button>\r\n    <div class=\"mobile-menu\">\r\n      <ul class=\"header-nav-list\">\r\n          {{#each cards}}\r\n              {{> categoryListItem this}}\r\n          {{/each}}\r\n        </ul>\r\n    </div>\r\n</nav>\r\n</div>
===================================================================
diff --git a/templates/partials/sidebar.hbs b/templates/partials/sidebar.hbs
--- a/templates/partials/sidebar.hbs	
+++ b/templates/partials/sidebar.hbs	
@@ -5,7 +5,7 @@
     </button>
     <div class="mobile-menu">
       <ul class="header-nav-list">
-          {{#each cards}}
+          {{#each categories}}
               {{> categoryListItem this}}
           {{/each}}
         </ul>
